collectors/okta/okta_collector.js

```diff
@@ -60,7 +61,13 @@ class OktaCollector extends PawsCollector {
             return callback(null, logAcc, newState, newState.poll_interval_sec);
         })
         .catch((error) => {
-            return callback(error);
+            if (error.message && error.message.match(THROTTLING_ERROR_REGEXP)) {
```

Comment:
Just for completeness, is there no status code for this sort of error? Do we need to do a match on this?

Reply:
Nope. SDK simply throws `new Error` in cae of throttling https://github.com/okta/okta-sdk-nodejs/blob/4643cf821c2aa77ecefbf92ddcf24e7c0b068862/src/default-request-executor.js#L93-L96

Reply:
Fun. Ok sounds good to me.